remo/profiles/fixtures/demo_functional_areas.json

```diff
@@ -17,15 +17,6 @@
             "slug": "addons"
         }
     },
-    {
```

Comment:
Can you please order these in an ascending order by `prk`? Can you also change those in order to be consecutive?